### ISSUE TITLE
Color initialization from Hsl instance screws up @rgb value

### ISSUE THREAD

**Issue by jfairbank** [jfairbank/chroma#11]
``` ruby
'red'.paint.to_hsl_s                  #=> hsl(0, 100%, 50%)
Chroma::Color.new('red'.paint.to_hsl) #=> hsl(0, 1%, 1%)
```

**Comment by jfairbank** [comment]
This applies to Hsv too.

**Comment by jfairbank** [comment]
Fixed by ce3cc3f.
